main_example_problem.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from test_example_problem import tip_calculator
+
+def main():
+    """Calculate tip and total bill with tip"""
+
+# inputs for function
+    bill_amount = float(input("What is the bill? "))
+    tip_percent = float(input("What is the tip percent? "))
+
+# call function with variables
+    tip_calculation = tip_calculator(bill_amount, tip_percent)
+
+# coordinate output
+    bill_with_tip = bill_amount + tip_calculation
```

Comment:
This is something that can also be moved into a function `calculate_total_bill`, passing in the `bill_amount` and `tip`.